configuration/pipeline_config.json

```diff
@@ -5,7 +5,7 @@
       "SourceType": "RapidPro",
       "Domain": "textit.in",
       "TokenFileURL": "gs://avf-credentials/imaqal-text-it-token.txt",
-      "ContactsFileName": "ssf_rec_contacts",
+      "ContactsFileName": "ssf_dcf_contacts",
```

Comment:
rename the local file in Miranda too so that you don't have to fetch from beginning again when you merge this

Reply:
Noted, thanks.